tests/console/commands/test_add.py

```diff
@@ -357,7 +357,7 @@ def test_add_directory_constraint(
     mocker: "MockerFixture",
 ):
     p = mocker.patch("pathlib.Path.cwd")
-    p.return_value = Path(__file__).parent
+    p.return_value = Path(__file__).parent.parent.parent / "fixtures" / "git"
```

Comment:
This parent calls are a bit messy, but also I think we should address them as part of a 'fixtures-to-get-fixtures' refactor to the test suite.

Reply:
note that you can also use [`Path.parents`](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.parents):

```
    p.return_value = Path(__file__).parents[2] / "fixtures" / "git"
```